pageserver/src/tenant/timeline.rs

```diff
@@ -1872,8 +1876,7 @@ impl Timeline {
                     }
                     (DownloadBehavior::Error, false) => {
                         return Err(PageReconstructError::NeedsDownload(
-                            timeline.myself.clone(),
-                            Arc::downgrade(&remote_layer),
+                            remote_layer.file_name.clone(),
```

Comment:
We're missing the timeline information from the error now. Not a big deal, but it would be good to include the timeline in the "layer {} needs download" message. The missing layer could be on an ancestor timeline.

Reply:
Did not consider ancestors when decided that it's ok to avoid timeline id here.
Now printing `{tenant_id}/{timeline_id}/{layer_name}` in logs.